requirements/base.txt

```diff
@@ -41,7 +41,6 @@ suds==0.4
 tinys3==0.1.11
 unipath>=1.1,<=2.0
 vobject==0.9.1
-wagtail==1.8.1
```

Comment:
Thanks. Also, is it worth using version [1.10.1](https://pypi.python.org/pypi/wagtail/1.10.1) (latest on PyPI right now) for this PR? [Release notes are here](http://docs.wagtail.io/en/v1.10.1/releases/1.10.1.html), looks like just minor bug fixes.